.travis.yml

```diff
@@ -6,6 +6,7 @@ script:
 notifications:
   email: false
 rvm:
+  - 2.5.1
```

Comment:
I think we need to add either 2.4.0 or 2.5.1 to the exclusion matrix below, otherwise we'll end up testing commits, warnings, etc on 2 ruby platforms.